planning/autoware_behavior_path_static_obstacle_avoidance_module/src/scene.cpp

```diff
@@ -915,11 +917,21 @@ BehaviorModuleOutput StaticObstacleAvoidanceModule::plan()
     ignore_signal_ = is_ignore ? std::make_optional(uuid) : std::nullopt;
   };
 
+  const auto is_large_deviation = [this](const auto & path) {
+    constexpr double THRESHOLD = 1.0;
```

Comment:
clang-tidy will trigger a readability warning due to the variable name being all upper case.
probably better to use lowercase to remove the warning.

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/7045/commits/ef6b54cfef55d22b88cbca2fa7bf55654b85fbde